submissions/5mountains/popup/css/main.css

```diff
@@ -0,0 +1,291 @@
+    width: 35px;
+    height: 35px;
+    border-radius: 50%;
+}
+
+.menu__link-user:hover {
+    box-shadow: 0 1px 0 rgba(0,0,0,.15);
+}
+/* 
+    Main menu style END
+*/
+
+
+
+/* 
+    Submenu of main menu style START
+*/
+.menu__submenu {
+    z-index: 5;
+    width: 320px;
+    max-height: 500px;
+    position: absolute;
+    display: none;
+    top: 65px;
+    right: 70px;
+    border: 1px solid #ccc;
+    z-index: 1;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    overflow-x: hidden;
+    padding: .5rem .5rem 1.6rem .5rem;
+    box-shadow: 3px 3px 10px rgba(0, 0, 0, .3);
+}
+
+.submenu__item {
+    height: 100px;
+    width: 100px;
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 1rem;
+}
+
+.submenu__link {
+    width: 100%;
+    height: 100%;
+    display: block;
+    text-align: center;
+    border: 1px solid transparent;
+}
+
+.submenu__link img {
+    width: 75%;
+}
+
+.submenu__link:hover {
+    border: 1px solid #ccc;
+}
+
+.submenu__link:focus {
+    border: 1px solid #00f; 
+}
+
+.menu__submenu [type="checkbox"] {
+    display: none;
+}
+
+/* scrollbar style start*/
+.menu__submenu::-webkit-scrollbar {
+    width: .15rem;
+}
+
+.menu__submenu::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 5px rgba(0, 0, 0, 0.3);
+}
+
+.menu__submenu::-webkit-scrollbar-thumb {
+    background-color: rgb(182, 180, 180);
+    border: 1px solid rgb(58, 64, 70);
+}
+/* scrollbar style end*/
+
+.submenu__item-label {
+    position: absolute;
+    width: 100%;
+    bottom: 0;
+    left: 0;
+    background: #f1f1f1;
+    cursor: pointer;
+    padding: .4rem 0;
+    text-align: center;
+}
+/* 
+    Submenu of main menu style END
+*/
+
+
+/*
+    Extra menu of submenu style START
+*/
+.submenu__extramenu {
+    display: none;
+    flex-wrap: wrap;
+    overflow: auto;
+    margin-top: 3rem;
+}
+
+.submenu__item-input:checked ~ .submenu__extramenu {
+    display: flex;
+}
+
+.submenu__item-input:checked ~ .submenu__item-label {
+    display: none;
+} 
+
+.submenu__item-input:checked .menu__submenu {
+    overflow: auto;
+}
+
+.submenu__extramenu::before{
+    z-index: 9;
+    content: '';
+    width: 100%;
+    position: absolute;
+    top: 490px;
+    left: 0;
+    border-bottom: 1px solid #ccc;
+}
+
+.extramenu__item {
+    height: 100px;
+    width: 100px;
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 1rem;
+}
+
+.extramenu__link {
+    width: 100%;
+    height: 100%;
+    display: block;
+    text-align: center;
+    border: 1px solid transparent;
+}
+
+.extramenu__link:hover {
+    border: 1px solid #ccc;
+}
+  
+.extramenu__link:focus {
+    border: 1px solid #00f;
+}
+
+.extramenu__link img {
+    width: 75%;
+}
+
+.extramenu__extralink {
+    width: 100%;
+    padding-top: 1.5rem;
+    display: block;
+    text-align: center;
+}
+
+.extramenu__extralink:hover {
+    text-decoration: underline;
+    opacity: .75;
+}
+/* 
+    Extra menu of submenu of main menu style END
+*/
```

Comment:
Add empty new line

Reply:
I will know that a new blank line between comments and code is mandatory! I thought it was better for clarity (:

Reply:
I meant that you should always leave a blank line at the end of the file.

Reply:
Okey! I will!